honeybread-core/src/main/java/com/whatsub/honeybread/core/domain/store/StoreRepositoryImpl.java

```diff
@@ -0,0 +1,38 @@
+package com.whatsub.honeybread.core.domain.store;
+
+import com.querydsl.core.types.dsl.BooleanExpression;
+import com.whatsub.honeybread.core.domain.store.dto.StoreSearch;
+import com.whatsub.honeybread.core.support.QuerydslRepositorySupport;
+import org.apache.commons.lang3.StringUtils;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.Pageable;
+
+import static com.whatsub.honeybread.core.domain.store.QStore.store;
+
+public class StoreRepositoryImpl extends QuerydslRepositorySupport implements StoreRepositoryCustom {
+
+    public StoreRepositoryImpl() {
+        super(Store.class);
+    }
+
+    @Override
+    public Page<Store> getStores(final Pageable pageable, final StoreSearch search) {
+        return applyPagination(pageable, query -> query.from(store)
+                .where(likeName(search.getName()),
+                        equalSellerId(search.getSellerId()),
+                        equalStoreStatus(search.getStatus())));
```

Comment:
괄호가 너무 많아서 어디까지가 메소드 조건절 호출인지 알아보기가 힘든것 같아요
개인적으론 이런 스타일을 좋아합니다
```suggestion
        return applyPagination(pageable, 
            query -> query
                .from(store)
                .where(
                    likeName(search.getName()),
                    equalSellerId(search.getSellerId()),
                    equalStoreStatus(search.getStatus())
                )
        );
```